packages/rancher-webhook/package.yaml

```diff
@@ -0,0 +1,2 @@
+url: https://github.com/rancher/webhook/releases/download/v0.2.4-rc7/rancher-webhook-0.2.4-rc7.tgz
+version: 1.0.3
```

Comment:
Can you cut a 0.2.5-rc1 with actual new code in master, then update this in charts dev-v2.6 to be 1.0.4+0.2.5-rc1, then announce somewhere so people don't accidentally use 0.2.4.

Reply:
Will do, do you want the `v0.2.5-rc1` tag to be on the latest commit in master?

Reply:
If yes, then the latest commit will have two tags: `v0.2.4-rc6` and `v0.2.5-rc1`, if that's OK...

Reply:
Yes thats ok. We want 0.2.4-rc6 to be forgotten.